src/components/Button.jsx

```diff
@@ -1,35 +1,160 @@
   a,
   button {
     text-decoration: none;
     outline: none;
-    color: #001426;
+    color: var(--blizzard-black);
     cursor: pointer;
+    svg {
+      margin-right: -0.25rem;
+      padding: 0.12rem;
+    }
   }
   .primary {
-    background-color: red;
-    border: 2px solid black;
-    width: 6.5rem;
-    height: 2.4rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0.5rem 1rem;
+    width: 7rem;
+    height: 2.3rem;
+    background: var(--white);
+    border: 3px solid var(--dark-red);
+    box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 2.2px;
+    &.small {
+      width: 6rem;
+      height: 2rem;
+    }
+    svg {
+      fill: red;
+    }
+  }
+  .primary,
+  .primary.small {
+    &:hover {
+      color: var(--white);
+      background: var(--dark-red);
+      box-shadow: 2px 3px 4px var(--shadow-black);
+      border-radius: 2.2px;
+      svg {
+        fill: var(--white);
+      }
+    }
+    &:active {
+      background: var(--darker-red);
+      box-shadow: none;
+    }
+    &:focus {
+      border: 2px solid var(--white);
+      outline: 2px solid var(--medium-blue);
+      border-radius: 2.2px;
+      background: var(--dark-red);
+      color: var(--white);
+      svg {
+        fill: var(--white);
+      }
+    }
+    &:disabled {
+      color: var(--dark-grey);
+      background: var(--light-grey);
+      border: none;
+      box-shadow: none;
+      svg {
+        fill: var(--dark-grey);
+      }
+    }
   }
   .secondary {
-    background-color: green;
-    width: 8rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0.5rem 1rem;
+    width: 5rem;
+    height: 2.31rem;
+    background: var(--white);
+    border: 2px solid var(--blizzard-blue);
+    box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.25);
+    border-radius: 2.2px;
+    &.small {
+      width: 4.63rem;
+      height: 2.13rem;
+    }
+  }
+  .secondary,
+  .secondary.small {
+    &:hover {
+      box-shadow: 1px 2px 6px rgba(0, 0, 0, 0.6);
+    }
+    &:active {
+      background: var(--blizzard-blue);
+      box-shadow: none;
+    }
+    &:focus {
+      background: var(--white);
+      border: 1.5px solid var(--white);
+      border-radius: 3.3px;
+      outline: 2px solid var(--medium-blue);
+      box-shadow: inset 0 0 3px var(--medium-blue);
+    }
+    &:disabled {
+      border: 2px solid var(--light-grey);
+      color: var(--dark-grey);
+    }
   }
   .textBtn {
+    width: 8.5rem;
+    height: 1.31rem;
+    display: flex;
+    justify-content: left;
+    align-items: center;
+    padding: 0px;
+    border: none;
+    color: var(--dark-red);
+    border-radius: 2.2px;
+    background: none;
+    text-decoration: none;
+    &.small {
+      height: 1.12rem;
+      width: 5.75rem;
+    }
   }
-  .small {
-    width: 4rem;
-    height: 2rem;
+  .textBtn,
+  .textBtn.small {
+    transition: padding-left 250ms ease;
```

Comment:
So close. I just didn't give you enough info. So the exact target we want to transition is `.textBtn > img`.

So if we move this transition declaration to target that, it should work. Or since this is SASS syntax all we need to do is nest an `img` here. And then line 130 will change the padding-left of the direct img descendent of the .textBtn class.

And then I think we'll be good.

Reply:
Ok sounds good